src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard/dashboard.component.html

```diff
@@ -3,7 +3,18 @@
     <cd-card title="Details"
              i18n-title
              class="col-sm-3 px-3">
-             Text
+      <table *ngFor="let item of detailsCardData | keyvalue:zero">
```

Comment:
```suggestion
      <table *ngFor="let detailsCardItem of detailsCardData | keyvalue:zero">
```
or something more meaningful?

Reply:
Sure. I'll change it.